maestro-ios/src/main/java/ios/simctl/Simctl.kt

```diff
@@ -181,15 +189,26 @@ object Simctl {
     }
 
 
-    fun launch(deviceId: String, bundleId: String) {
+    fun launch(deviceId: String, bundleId: String, language: String?) {
+        var command = mutableListOf(
+            "xcrun",
+            "simctl",
+            "launch",
+            deviceId,
+            bundleId,
+        )
+
+        language?.let {
+            if (supportedLanguages.contains(it)) {
```

Comment:
Are we throwing exceptions if someone makes an invalid selection?

Reply:
We should certainly inform user that their language selection was invalid